src/tracker.jl

```diff
@@ -0,0 +1,34 @@
+abstract type Tracker end
+
+function init_tracker(tracker::Tracker, image::Array{T, 2}, bounding_box::MVector{4, Int}) where T
+    @assert bounding_box[1] >= 1 && bounding_box[1] <= size(image)[1]
+    @assert bounding_box[2] >= 1 && bounding_box[2] <= size(image)[2]
+    @assert bounding_box[3] >= 1 && bounding_box[3] <= size(image)[1]
+    @assert bounding_box[4] >= 1 && bounding_box[4] <= size(image)[2]
+
+    @assert bounding_box[1] < bounding_box[3]
+    @assert bounding_box[2] < bounding_box[4]
```

Comment:
speaking of `bouding_box`, it might be better to start with a more general type
```julia
abstract type RegionOfInterest
struct BoxROI <: RegionOfInterest
    img
    bounds
end
```
and move these assertions to the constructor.

I know you don't need/plan to implement other types of ROI here, but by doing this you can ease your/others' future development.

Reply:
Ok, I personally like this style and surely will make changes accordingly. Thank you so much!